global.json

```diff
@@ -3,6 +3,8 @@
     "dotnet": "8.0.100-preview.7.23376.3"
   },
   "msbuild-sdks": {
+    "Microsoft.Build.NoTargets": "3.5.0",
```

Comment:
It doesn't seem like any of the projects in this repo use `Microsoft.Build.NoTargets` explicitly. How does it get used?

Reply:
Oh. Probably doesn't. I pulled this from another repo, to fix the "no SDK specifier in .proj means nuget restore doesn't happen" problem. At some point that repo was probably using NoTargets.